src/cloud_common/protocols/unit/alert.py

```diff
@@ -0,0 +1,339 @@
+#
+#  Copyright (c) 2018-2024 Renesas Inc.
+#  Copyright (c) 2018-2024 EPAM Systems Inc.
+#
+from datetime import datetime
+from typing import Annotated, Literal, Union
+
+from pydantic import BaseModel, Discriminator, Field
+
+from cloud_common.protocols.unit.constants import DataSizes
+from cloud_common.protocols.unit.types import (
+    TypeAlertMessageMandatory,
+    TypeCoreComponentIdMandatory,
+    TypeDeviceMandatory,
+    TypeNodeIdMandatory,
+    TypeVersionMandatory,
+)
+
+# AlertTagSystemError      = "systemAlert"            # noqa: E800
+# AlertTagAosCore          = "coreAlert"              # noqa: E800
+# AlertTagResourceValidate = "resourceValidateAlert"  # noqa: E800
+# AlertTagDeviceAllocate   = "deviceAllocateAlert"    # noqa: E800
+# AlertTagSystemQuota      = "systemQuotaAlert"       # noqa: E800
+# AlertTagInstanceQuota    = "instanceQuotaAlert"     # noqa: E800
+# AlertTagDownloadProgress = "downloadProgressAlert"  # noqa: E800
+# AlertTagServiceInstance  = "serviceInstanceAlert"   # noqa: E800
+
+
+class AosBaseAlert(BaseModel):
+    timestamp: Annotated[
+        datetime,
+        Field(
+            alias='timestamp',
+            description='Timestamp when alert was triggered in ISO8601 format',
+        ),
+    ]
+
+
+class AosAlertSystemError(AosBaseAlert):
+    """Aos Unit system error alert information."""
+
+    tag: Annotated[
+        Literal['systemAlert'],
+        Field(description='type of the alert'),
+    ]
```

Comment:
Do you follow any rule to put empty line between class fields?

Reply:
https://peps.python.org/pep-0257/#multi-line-docstrings
https://peps.python.org/pep-0008/#blank-lines

A few words: blank line after docstring
Blank lines to logical grouping

Reply:
I see that sometimes you put empty line after multiline field, sometimes no:

```python
   tag: Annotated[
        Literal['coreAlert'],
        Field(description='type of the alert'),
    ]
    node_id: TypeNodeIdMandatory
    core_component: TypeCoreComponentIdMandatory
    message: TypeAlertMessageMandatory
```

```python

    instance: Annotated[
        int,
        Field(
            description='Instance number (starting from 0)',
        ),
    ]

    version: TypeVersionMandatory
    message: TypeAlertMessageMandatory
```

Use one pattern.